dpe/src/commands/rotate_context.rs

```diff
@@ -17,6 +17,14 @@ pub struct RotateCtxCmd {
     target_locality: u32,
 }
 
+impl RotateCtxCmd {
+    pub const TARGET_IS_DEFAULT: u32 = 1 << 31;
+    
+    const fn uses_target_is_default(&self) -> bool {
```

Comment:
optional: maybe a more concise name would be `makes_default`

Reply:
Or `make_default`

Reply:
Reason I did uses_target_is_default is to maintain consistency with flags in derive_child since we are using similar naming schemes there.